ciao-sdk/trace.go

```diff
@@ -0,0 +1,31 @@
+package sdk
+
+import (
+	"github.com/ciao-project/ciao/ciao-controller/types"
+	"github.com/ciao-project/ciao/client"
+	"github.com/pkg/errors"
+)
+
+func GetTraces(c *client.Client, flags CommandOpts) (types.CiaoTracesSummary, error) {
+	traces, err := c.ListTraceLabels()
+	if err != nil {
+		return traces, errors.Wrap(err, "Error listing trace labels")
+	}
+
+	return traces, err
+}
+
+func GetTraceData(c *client.Client, flags CommandOpts) (types.CiaoTraceData, error) {
+	var traceData types.CiaoTraceData
+
+	if flags.Args[0] == "" {
```

Comment:
Are we sure there'll always be at leas one argument if we get here?

Reply:
Yes it's guaranteed since it will only be called when Show() is given some argument(s). If we're trying to future proof it for use in other code paths I can add the check in here as well, but afaik that's not the plan.